package.json

```diff
@@ -13,7 +13,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "semver": "^7.3.5",
-    "swingset": "^0.10.1"
+    "swingset": "^0.11.0"
```

Comment:
Not sure why this is marked as different from `main`, from what I can tell, `main` is already on `swingset^0.11.0`.